### ISSUE TITLE
Merge gsoc-develop to develop

### ISSUE THREAD

**Issue by smcgregor** [privly/privly-chrome#132]
A few outstanding issues before we can merge into develop:

* [Update deprecated messenger](https://github.com/privly/privly-chrome/pull/129/files#r36582808)
* [Update comments](https://github.com/privly/privly-chrome/pull/129/files#r36588912)
* [Edge case](https://github.com/privly/privly-chrome/pull/129/files#r36588978)
* [nits like these](https://github.com/privly/privly-chrome/pull/129/files#r36582901)

**Comment by smcgregor** [comment]
Another issue: the icon turns green when using the seamless posting form, but it doesn't change away from green when the input context leaves the posting iframe.

**Comment by smcgregor** [comment]
Another issue: The posting button won't appear on issues of the privly-chrome repository. I think there is a magic string somewhere that checks for privly-chrome in the path?

**Comment by SummerWish** [comment]
> Another issue: the icon turns green when using the seamless posting form, but it doesn't change away from green when the input context leaves the posting iframe.

I think I have fixed this in previous commits and it should works in most of the cases. Could you open individual issue for this if there are problems in certain conditions?

**Comment by SummerWish** [comment]
> Another issue: The posting button won't appear on issues of the privly-chrome repository. I think there is a magic string somewhere that checks for privly-chrome in the path?

Fixed in this PR: https://github.com/privly/privly-applications/pull/285

**Comment by SummerWish** [comment]
Issues listed here are fixed in latest commits in this PR: https://github.com/privly/privly-chrome/pull/129

**Comment by smcgregor** [comment]
I'll open a new issue if the problem persists after I merge all your more recent changes.
